lib/ddtrace/contrib/rails/core_extensions.rb

```diff
@@ -1,50 +1,124 @@
 module Datadog
   # RailsRendererPatcher contains function to patch Rails rendering libraries.
+  # rubocop:disable Lint/RescueException
```

Comment:
it suggests me to catch `StandardError` instead of `Exception` where we re-raise